package.json

```diff
@@ -88,72 +90,60 @@
     "babel-loader": "^8.0.5",
     "babel-plugin-dev-expression": "^0.2.2",
     "babel-plugin-dynamic-import-node": "^2.2.0",
+    "babel-plugin-inline-react-svg": "^1.1.0",
     "babel-plugin-lodash": "^3.3.2",
     "babel-plugin-module-resolver": "^3.2.0",
     "babel-plugin-react-docgen": "^3.0.0",
     "codecov": "^3.2.0",
+    "commitizen": "^3.1.2",
     "component-playground": "^3.2.1",
-    "cross-env": "^5.2.0",
+    "cross-env": "^6.0.3",
+    "css-loader": "^3.2.0",
+    "cz-customizable": "^6.2.0",
+    "cz-customizable-ghooks": "^1.5.0",
     "docz": "^1.2.0",
     "docz-core": "^1.2.0",
     "docz-theme-default": "^1.2.0",
     "eslint-plugin-prettier": "^3.1.0",
     "expect": "^24.5.0",
     "gh-pages": "^2.0.1",
     "glob": "^7.1.3",
-    "husky": "^1.3.1",
+    "husky": "^3.0.8",
```

Comment:
`husky`, `prettier` and `prettier-eslint` are all included in `@sumup/foundry` and shouldn't be listed as direct dependencies.